src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -51,7 +51,7 @@
     "d:transactions/measurements.frames_total@none": PREFIX + 113,
     "d:transactions/measurements.stall_count@none": PREFIX + 114,
     "d:transactions/measurements.stall_longest_time@millisecond": PREFIX + 115,
-    "d:transactions/measurements.stall_percentage@percent": PREFIX + 116,
+    "d:transactions/measurements.stall_percentage@ratio": PREFIX + 116,
```

Comment:
I verified that neither the old nor the new version of this string exist in either `sentry_stringindexer` or `sentry_perfstringindexer`.